.github/workflows/update-schema.yml

```
@@ -74,11 +74,36 @@ jobs:
           git_user_signingkey: true
           git_commit_gpgsign: true
 
-      - name: Copy updated schema
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Install Gomplate
+        run: brew install gomplate
```

I'm not a fan of not fixing the versions, it poses problems of reproducibility. What I can suggest is that to reuse the recipe from the repo [okp4/networks](https://github.com/okp4/networks/blob/70ae602950ebf4bafaca7fb37681f3d3944573bb/.github/workflows/update-chains-description.yml#L58).

```suggestion
      - name: Install gomplate
        run: |
          curl -s -L -o /usr/local/bin/gomplate https://github.com/hairyhenderson/gomplate/releases/download/v3.11.5/gomplate_linux-amd64
          chmod +x /usr/local/bin/gomplate
```

---

Yes absolutely !